### DISCUSSION TITLE
Android 源码分析 —— 从 Toast 出发 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#237] [Announcements]
https://mazhuang.org/2017/11/12/start-from-toast/ 

 本系列文章在 https://github.com/mzlogin/rtfsc-android 持续更新中，欢迎有兴趣的童鞋们关注。

**Comment by helinyu** [comment]
问一下，你的博客是使用哪个框架搭建的？

**Comment by mzlogin** [comment]
> @helinyu
> 问一下，你的博客是使用哪个框架搭建的？

是托管的 GitHub Pages，基于 Jekyll。源码在 <https://github.com/mzlogin/mzlogin.github.io>
